src/ansys/fluent/core/solver/error_message.py

```diff
@@ -25,3 +25,55 @@ def allowed_values_error(
     context: str, trial_name: str, allowed_values: List[str]
 ) -> ValueError:
     return ValueError(allowed_name_error_message(context, trial_name, allowed_values))
+
+
+class FluentConnectionError(ValueError):
```

Comment:
Why does this inherit from `ValueError`?

Reply:
![image](https://github.com/ansys/pyfluent/assets/106588300/8aca75c2-4096-4d6c-b9bd-7bea46855e95)

https://stackoverflow.com/questions/39077623/can-a-python-inner-class-be-a-subclass-of-its-own-outer-class#:~:text=not%20a%20subclass.-,No%2C%20an%20inner%20class%20can't%20inherit%20(not%20extend,while%20defining%20the%20inner%20class.

Reply:
Right, we followed `TransactionError` format which includes all `RuntimeError`. it's removed now. Thank you.